codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSAuthUtils.kt

```diff
@@ -18,8 +18,11 @@ import software.amazon.smithy.swift.codegen.SwiftWriter
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
```

Comment:
Refactored to remove logic for getting auth schemes supported by the model and move it into `smithy-swift`. Refer to comment: https://github.com/smithy-lang/smithy-swift/pull/786/files#r1681867714